.github/workflows/build_and_deploy.yml

```
@@ -0,0 +1,65 @@
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}/app
+      IMAGE_TAG: ${{ github.sha }}
+    outputs:
+      commit_title: ${{ steps.get_head_commit_title.outputs.title }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - id: get_head_commit_title
+        run: echo "title=$(git log --format=%B -n 1 HEAD | head -n 1)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:62: Double quote to prevent globbing and word splitting

(shellcheck)